wally-registry-frontend/build.js

```diff
@@ -3,20 +3,30 @@
 // This script bundles the static site generator using Parcel, which in turn
 // will invoke Parcel again itself, reusing the same assets.
 
-const Parcel = require("parcel-bundler")
+const { Parcel } = require("@parcel/core")
 
 process.env.NODE_ENV = "production"
 
 const publicUrl = process.env.PARCEL_PUBLIC_URL || "/"
 
 console.log("Bundling client with Parcel...")
 
-const bundler = new Parcel(["src/index.html"], {
-  outDir: "build/static",
-  publicUrl: "/static",
-  watch: false,
-  autoInstall: false,
+const bundler = new Parcel({
+  entries: ["src/index.html"],
+  defaultConfig: "@parcel/config-default",
+  defaultTargetOptions: {
+    distDir: "build/static",
+    publicUrl: "/static",
+  },
   logLevel: 2,
 })
 
-bundler.bundle().then(() => {})
+// ,["src/index.html"], {
+//   outDir: "build/static",
+//   publicUrl: "/static",
+//   watch: false,
+//   autoInstall: false,
+//   logLevel: 2,
+// })
```

Comment:
Do you anticipate us needing this commented out section or can we just delete it?